### PR TITLE
JAVA-1642: Document metadata

### PR DESCRIPTION
**Pull request by olim7t**

### CODE REVIEW

File: manual/core/address_resolution/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Each node in the Cassandra cluster is uniquely identified by an IP address that the driver will use
+to establish connections.
+
+* for contact points, these are provided as part of configuring the `Cluster` object;
+* for other nodes, addresses will be discovered dynamically, either by inspecting `system.peers` on
+  already connected nodes, or via push notifications received on the control connection when new
+  nodes are discovered by gossip.
+
+
+### Cassandra-side configuration
```
Comment (tolbertam) [inline manual/core/address_resolution/README.md:12]: This section is very useful.  It's a common problem new users run into (not understanding why can't connect w/ driver but cqlsh works locally).

File: manual/core/metadata/schema/README.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+```
+
+Schema agreement is checked:
+ 
+* before a schema refresh;
+* before [repreparing all queries](../../statements/prepared#how-the-driver-prepares) on a newly up
+  node;
+* before completing a successful schema-altering query (like in our example above).
+
+It is done  by querying system tables to find out the schema version of all nodes that are currently
```
Comment (tolbertam) [inline manual/core/metadata/schema/README.md:158]: very small nit, there are two spaces between `done` and `by`

**Review by tolbertam — APPROVED** [review]
This is excellent!  Only noticed one spacing issue, but other than that this is perfect!

**Comment by olim7t** [comment]
Merged this one manually, for some reason the Github UI didn't update when I force-pushed the branch.

**Pull request closed without merging**